.github/workflows/deploy-infra.yaml

```diff
@@ -54,12 +54,18 @@ jobs:
             ERROR=true
           fi
 
+          if [ -z "${{ secrets.PRIMARY_DOMAIN_NAME }}" ] || [ -z "${{ secrets.DOMAIN_NAME }}" ]
+          then
+            echo "No domain name provided. Set PRIMARY_DOMAIN_NAME and DOMAIN_NAME in repository secrets."
+            exit 1
+          fi
```

Comment:
I would change the error message to be more accurate. Something like this:

PRIMARY_DOMAIN_NAME or DOMAIN_NAME is not set. Make sure that they are both set in repository secrets.

Or I would split the if into 2 ifs - one if for if PRIMARY_DOMAIN_NAME exists and one if for DOMAIN_NAME